docs/visual-basic/developing-apps/programming/app-settings/accessing-application-settings.md

```diff
@@ -34,7 +34,7 @@ translation.priority.ht:
   - "zh-cn"
   - "zh-tw"
 ---
-# Accessing Application Settings (Visual Basic)
+# Accessing application settings (Visual Basic)
```

Comment:
It might, but most of the topics in MSDN use this convention. It would be a major work item to change all of them; the purpose of this PR is simply to fix the links.